radio/src/targets/horus/tp_gt911.cpp

```diff
@@ -763,6 +764,12 @@ struct TouchState touchPanelRead()
             TRACE("I2C B1 ReInit failed");
         return internalTouchState;
       }
+        
+#if defined (PCBX12S)
+          touchData.points[0].x = LCD_W - touchData.points[0].x;
+          touchData.points[0].y = LCD_H - touchData.points[0].y;
```

Comment:
On above two lines, the indentation needs a tiny adjustment as well.

Reply:
Thanks, done!